lib/ViewOrganizationCard/ViewOrganizationCard.js

```diff
@@ -149,16 +149,22 @@ const ViewOrganizationCard = ({
     })
   );
 
-  const renderOrganizationName = (name) => (
-    <KeyValue label={<FormattedMessage id="stripes-erm-components.viewOrg.name" />}>
-      <span
-        data-test-org-name
-        id={`${id}-name`}
-      >
-        {name}
-      </span>
-    </KeyValue>
-  );
+  const renderOrganizationName = (name) => {
+    const orgStatus = org.orgsUuid_object?.status ? org.orgsUuid_object.status :
```

Comment:
@CalamityC For brevity, in future this can be written `org.orgsUuid_object?.status ?? org?.status`. Not going to block this merge for that though, it can be tweaked later if we want.